src/main/java/uk/gov/hmcts/ccd/v2/V2.java

```diff
@@ -43,6 +43,8 @@ private MediaType() {}
 
         public static final String CASE_EVENTS = "application/vnd.uk.gov.hmcts.ccd-data-store-api.case-events.v2+json;charset=UTF-8";
 
+        public static final String CASE_USER_ROLES = "application/vnd.uk.gov.hmcts.ccd-data-store-api.case-user-roles.v2+json;charset=UTF-8";
```

Comment:
We wouldn't use a new media type for the initial version of this operation.

Reply:
Not using any media type for now!!!